SimVascular/SimVascular
# Fix to dicom vti issues

This PR fixes the issue where if you load a dicom and convert it to .vti, the views are all changed the next time you reload the project.

The fix stores the initial indexToWorldTransform matrix when the dicom is first loaded. When reloading the project, SimVascular will look for the transform file and set the reloaded image's transform to be what was in the file.

Tested this with OSMSC0110 dicom from the sv website and with Stephanies dicom, fixed in both cases. But could probably use more test cases.

To test:
* Create project
* load dicom image
* Convert to .vti
* Create a path with a few points
* Save and close project
* Reopen
* Check if all views still the same, path in correct location
